core/templates/pages/exploration-player-page/services/answer-classification.service.ts

```diff
@@ -68,8 +63,9 @@ export class AnswerClassificationService {
   private classifyAnswer(
       answer: InteractionAnswer,
       answerGroups: AnswerGroup[],
-      defaultOutcome: Outcome,
-      interactionRulesService): AnswerClassificationResult {
+      defaultOutcome: Outcome | null,
```

Comment:
Why can this be `null`?

Reply:
https://github.com/oppia/oppia/blob/741cc401ec375d4bd148dd547a718db7bcdb7499/core/templates/domain/exploration/InteractionObjectFactory.ts#L74-L76